docs/source/installation/troubleshooting.rst

```diff
@@ -193,31 +195,29 @@ Follow the steps in `this StackExchange answer <https://superuser.com/questions/
 --------------------------------
 
 If ``choco install manimce`` failed,
-it is likely being caused by Python not being added to your ``PATH`` variable properly.
+it is because Python was not added to your ``PATH`` variable properly.
 Try running the following commands in your terminal:
 
 1. ``py --version``
 2. ``python --version``
 3. ``py3 --version``
 4. ``python3 --version``
 
-Minimally, ``py --version`` and ``python --version`` should return a version.
+Ideally, ``py --version`` and ``python --version`` should return a version.
```

Comment:
I think "minimally" here is used as "out of these four commands, at least the specified two should return a version, but the others could do so as well". Ideally sort of is a stronger, not necessarily intended meaning, no? Do you have any thoughts about that?

Reply:
Ah, that makes sense. I agree!